app/components/formatted_time.js

```diff
@@ -29,11 +29,8 @@ export default class FormattedTime extends React.PureComponent {
         } = this.props;
 
         if (timeZone) {
-            return intl.formatDate(moment.tz(value, timeZone).toDate(), {
```

Comment:
We were translating it before, why are we no longer translating it?

Reply:
@jespino the translations where only for PM/AM that is some languages is am/pm but is not massively important and the thing is that formatDate uses the underline `intl` library that does not have support for timeZones

Reply:
Ok if is on purpose, works for me :)

Reply:
Ok if is on purpose, works for me :)